doc/source/configuration.rst

```diff
@@ -125,6 +125,8 @@ The Nextflow configuration file is used to configure various settings for the *k
     singularity {
         enabled = true
         cacheDir = ''
+        autoMounts = true
+        envWhitelist = '' 
```

Comment:
envWhitelist also works as an option in the docker scope, so maybe we should add it there too.

Reply:
Good idea. What about adding in brackets "not tested"?

Reply:
I don't think it's necessary in this case.